content/posts/2020-08-15-foundation-announcement.kr.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "발표: 도지코인 재단의 재설립"
+date = "2021-08-16"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+크나큰 기쁨속에  **도지코인 재단** *(설립 2014)* 의 재설립을 발표합니다, 다시 회복한 집중으로 도지코인 이코 시스템, 커뮤니티의 지원, 도지코인 블록체인의 미래를 홍보할 것 입니다.
+
+### 초점
+
+도지코인 재단은 준수 도지코인 지갑의 기본 참조 구현으로 남아있는 '도지코인 코어' 프로젝트를 장악하기 위한 것이 아니다. 
```

Comment:
the original line is
"The Dogecoin Foundation is not here to ‘take control’ of the *Dogecoin Core Wallet* project, which remains the primary reference implementation of a compliant Dogecoin Wallet."

I have trouble understanding what compliant wallet means?